src/pages/index.js

```diff
@@ -204,10 +203,10 @@ const avatarPopupForm = new PopupWithForm(
       })
       .catch((error) => {
         console.error("Failed to update user avatar:", error);
+        avatarFormValidator.disableSubmitButton();
```

Comment:
`avatarFormValidator.disableSubmitButton();` should be in `then` (not in `catch`)